src/toisto/ui/text.py

```diff
@@ -210,3 +225,16 @@ def punctuated(text: str) -> str:
 def quoted(text: str, quote: str = "'") -> str:
     """Return a quoted version of the text."""
     return f"{quote}{text}{quote}"
+
+
+def enumerated(*texts: str, min_enumeration_length: int = 2) -> str:
+    """Return an enumerated version of the text."""
+    match len(texts):
+        case length if length > min_enumeration_length:
+            return enumerated(", ".join(texts[:-1]) + ",", texts[-1])
```

Comment:
Split into two lines:
```suggestion
            comma_separated_texts = ", ".join(texts[:-1]) + ","
            return enumerated(comma_separated_texts, texts[-1])
```